src/optional/optional_formatter.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+def summary(valobj, dict):
+	try:
+		if not (valobj.IsValid()):
+			return "<invalid>"
+
+		has_value = True if (valobj
+			.GetChildMemberWithName('storage_')
+			.GetChildMemberWithName('hasValue')
+			.GetValue()
+		) == 'true' else False
+		
+		if has_value:
+			return (valobj
+				.GetChildMemberWithName('storage_')
+				.GetChildMemberWithName('value')
+				# .GetValue()
+			)
+		else:
+			return "null"
```

Comment:
instead of null print "No value"
this will avoid confusions with the actual null pointer

Reply:
Done